docs/dev.rst

```diff
@@ -91,6 +91,17 @@ The Symbols Service webapp is at: http://localhost:3000
 How to
 ======
 
+How to set up a development container for VS Code
+-------------------------------------------------
+
+The repository contains configuration files to build a
+`development container <https://containers.dev/>`_ in the `.devcontainer`
+directory. If you have the "Dev Containers" extension installed in VS Code, you
+should be prompted whether you want to reopen the folder in a container on
+startup. You can also use the "Dev containers: Reopen in container" command
+from the command palette. The container has all Python requirements installed.
+IntelliSense, type checking, code formatting with Black and running the tests
```

Comment:
Nit: Maybe the line starting with "IntelliSense, ..." should be part of the previous sentence? Seems like it's an incomplete sentence on its own at the moment.

Reply:
This was meant to be a new sentence, but apparently I just forgot mid-sentence that I was writing a sentence.